com.enjoyingfoss.Parlera.yml

```diff
@@ -0,0 +1,33 @@
+app-id: com.enjoyingfoss.Parlera
```

Comment:
`enjoyinfoss.com` is 401 Permission denied. Is this under the control of the upstream project? (requirement for App-Id)

Reply:
Yes, I just haven't gotten around to publishing a Gitlab Pages site for it yet.

(I have noticed, though, that my previous app – FOSStriangulator – was published with the wrong address, with a .org instead of a .com . Is it possible to change it there if it's already being used?)

Reply:
Not without submitting it again and then marking the old one as EOL: https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

Reply:
@barthalion Alright, will do that next time.